platforms/hyperledger-fabric/configuration/roles/create/configtx/tasks/main.yaml

```diff
@@ -6,16 +6,27 @@
 # These tasks create the configtx.yaml file as the requirements mentioned in network.yaml 
 # file. The configtx.yaml file is consumed by the configtxgen binary to generate the 
 # genesis block and channels.
+
+############################################################################################
+# This task ensures that the directory exists, and creates it, if it does not exist
+- name: "Ensures {{ values_dir }}/{{ name }} dir exists"
+  file:
+    path: "{{ values_dir }}/{{ name }}"
+    recurse: yes
+    state: directory
+
 - name: "Remove old configtx file"
   file:
     path: "{{ config_file }}"
     state: absent
+  changed_when: false
```

Comment:
rather than changed_when false, I believe this should be no_idempotence, as the files are changing, it is just that we always regenerate the config file even when it exists because there is no way to know if anything changed in configuration since the last time.